hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/connecting.md

```diff
@@ -289,9 +289,8 @@ for buckets in the central and EU/Ireland endpoints.
 
 ```xml
 <property>
-  <name>fs.s3a.bucket.landsat-pds.endpoint.region</name>
+  <name>fs.s3a.bucket.us2w-dataset.endpoint.region</name>
```

Comment:
nit: typo, usw2-dataset (or let's just be clearer with `us-west-2-dataset`

Reply:
will do